aws-redis.yml

```diff
@@ -210,6 +210,14 @@ provision:
     prohibit_update: true
     default: ""
     details: The name of an existing snapshot to be restored into this new instance
+  - field_name: preferred_azs
+    type: array
+    prohibit_update: true
+    default: []
+    details: |
+      List of EC2 availability zones in which the nodes will be created.
+      The first item in the list will be the primary node.
+      Number of entries must equal to node_count.
```

Comment:
Good catch :).
Thanks.

Reply:
Do we get an error when the number of entries and node_count doesn't match? Do we want to test it? Just curious.

Reply:
I've tested it and you get an error saying that they must match.